api/database/FormsDOM.py

```diff
@@ -62,6 +62,7 @@ def getParentID(id):
 def updateFormData(id, data):
     writeR = dict(mongo.db.forms.update({'_id': ObjectId(id)}, {'$set': {'form_data': data}}))
     writeR = dict(mongo.db.forms.update({'_id': ObjectId(id)}, {'$set': {'last_updated': datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S")}}))
+    mongo.db.forms.update({'_id': ObjectId(id)}, {'$set': {'completed': True}})
```

Comment:
Can this be combined to one set

Reply:
I will combine it into one. It doesn't look like it changes much I just didn't know the use of writeR['nModified'] that is in line 66 and didn't waNt to mess it up. Let me know if it actually affects something.

Reply:
Shouldn't affect -- the return is to just make sure one is found and written to. Combining actually would make the return more meaningful because in the previous version, the first `writeR` is overwritten